src/common/telemetry_ops/requests_telemetry.rs

```diff
@@ -13,13 +13,15 @@ pub type HttpStatusCode = u16;
 
 #[derive(Serialize, Deserialize, Clone, Default, Debug, JsonSchema)]
 pub struct WebApiTelemetry {
-    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    // #[serde(skip_serializing_if = "HashMap::is_empty")]
+    // TODO: Refactor this as an Option
     pub responses: HashMap<String, HashMap<HttpStatusCode, OperationDurationStatistics>>,
 }
 
 #[derive(Serialize, Deserialize, Clone, Default, Debug, JsonSchema)]
 pub struct GrpcTelemetry {
-    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    // #[serde(skip_serializing_if = "HashMap::is_empty")]
+    // TODO: Refactor this as an Option
```

Comment:
Why are you suggesting an `Option` here? I don't think there's much to that as the `HashMap` can also be empty.

Reply:
I believe it's a OpenAPI problem